src/containers/AccountModal/AccountModal.tsx

```diff
@@ -60,6 +61,20 @@ const AccountModal = () => {
     }
   }, [viewParam, loading, isPublicView, user, toLogin]);
 
+  // handle websocket notifications
+  useEffect(() => {
+    if (user) {
+      const handleNotification = async () => {
+        if (user) {
+          await subscribeToNotifications(user.uuid, navigate);
+        }
+      };
+      handleNotification();
+    }
+    // we don't need to subscribe on each navigate change
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [user]);
```

Comment:
When combining @dbudzins suggestion, you can shorten this to:

```suggestion
  useEffect(() => {   
    if (user) subscribeToNotifications(user.uuid, navigate);
    // we don't need to subscribe on each navigate change
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [user]);
```

I don't know if this should be a custom hook or `useEffect`. A custom hook would be nicer, and it would be totally fine if it doesn't return anything;

```tsx
useAccountSubscriptions(user, navigate);
```

I do have worries that this change also affects the Cleeng integration. This part is really InPlayer specific.

Are update payment details still working after this change?

Reply:
This shouldn't affect Cleeng at all, it's for the websocket notifications, where cleeng's method only returns true and doesn't do anything else.